### PR TITLE
newsletter 9

### PR DESCRIPTION
**Pull request by maxandersen**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc **

### CODE REVIEW

File: _data/publications.yaml [hunk 1]
```diff
@@ -1,75 +1,88 @@
...
+- url: "https://antoniogoncalves.org/2021/01/12/merry-chistmas-and-happy-fallback-with-microprofile"
   date: "2021-01-12"
+  note: "Build microservices to help Santa schedule his deliveries using Quarkus and\
+    \ MicroProfile."
   author: "Antonio Goncalves"
-  note: "Build microservices to help Santa schedule his deliveries using Quarkus and MicroProfile."
-  url: "https://antoniogoncalves.org/2021/01/12/merry-chistmas-and-happy-fallback-with-microprofile"
+  id: "MerryChristmasHappyFallback-2021-01-12"
+  type: "article"
+  title: "Merry Chistmas and Happy Fallback with Microprofile"
```
Comment (gastaldi) [inline _data/publications.yaml:9]:
```suggestion
  title: "Merry Christmas and Happy Fallback with Microprofile"
```

Comment (maxandersen) [reply _data/publications.yaml:9]: but..thats what the title is ... :)

**Comment by maxandersen** [comment]
added last added story and some fixes to the script for managing publicatoins.

File: _data/publications.yaml (hunk 1 above)
Comment (gastaldi) [reply _data/publications.yaml:9]: LOL, you're right. Just clicked on the link and saw it. /cc @agoncal

Comment (agoncal) [reply _data/publications.yaml:9]: Oups, even the URL has the typo (`merry-chistmas-`)

Comment (maxandersen) [reply _data/publications.yaml:9]: I'm not touching it. It looks beautiful to me as is :)

**Comment by maxandersen** [comment]
merged manually

**Pull request closed without merging**